### ISSUE TITLE
Post-actions chmod - No such file

### ISSUE THREAD

**Issue by CumpsD** [dotnet/templating#1804]
Given:
https://github.com/dotnet/templating/wiki/Post-Action-Registry#change-file-permissions

```json
  "postActions": [
  {
    "condition": "(OS != \"Windows_NT\")",
    "description": "Make build script executable",
    "manualInstructions": [ { "text": "Run 'chmod +x build.sh'" }  ],
    "actionId": "cb9a6cf3-4f5c-4860-b9d2-03a574959774",
    "args": {
      "+x": "build.sh"
    },
    "continueOnError": true
  }]
```

I get as output:
```bash
Processing post-creation actions...
No such file or directory
```

But the file is there:

```bash
/d/T/template-test  la                                                                                                                                                                                                                   1242ms  Tue 2019-03-26 // 16:58
total 1.0M
drwxr-xr-x 1 cumpsd docker  512 Mar 26 16:58 ./
drwxr-xr-x 1 cumpsd docker  512 Mar 26 10:46 ../
-rw-r--r-- 1 cumpsd docker  304 Mar 26 16:58 build.sh
```

Anybody any ideas on how to debug this?

**Comment by mlorbetske** [comment]
It looks like this is a duplicate of #1743 - could you confirm please?

**Comment by CumpsD** [comment]
It is similar, except it is in WSL instead of OSX. And the permissions do not get applied at all, as opposed to the case on OSX where it apparently does at least get applied.

**Comment by donJoseLuis** [comment]
1. Adding item to specific project corresponding to a targeted delivery area
1. Once this has been done for the entire backlog, the next steps will be to prioritize within each project & output a delivery roadmap.
1. At that point, the issues will be assigned to a corresponding milestone
1. The team will then begin to deliver against the road map, contributions always welcome

**Comment by AR-May** [comment]
Hello @CumpsD! Is this still an issue? 
If yes, could you please provide the dotnet version (`dotnet --version`)?

**Comment by AR-May** [comment]
Closing this issue. It is my understanding that this was fixed within issue #2072. Please do not hesitate to reopen if I am wrong.
